inference-engine/tests/functional/inference_engine/serialization/single_layer/comparison_ops.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_layer/comparison.hpp"
+
+using namespace LayerTestsDefinitions;
+using namespace LayerTestsDefinitions::ComparisonParams;
+
+static const std::map<std::vector<size_t>, std::vector<std::vector<size_t>>> inputShapes = {
+    {{1}, {{1}, {17}, {1, 1}, {2, 18}, {1, 1, 2}, {2, 2, 3}, {1, 1, 2, 3}}},
+    {{5}, {{1}, {1, 1}, {2, 5}, {1, 1, 1}, {2, 2, 5}}},
+    {{2, 200}, {{1}, {200}, {1, 200}, {2, 200}, {2, 2, 200}}},
+    {{1, 3, 20}, {{20}, {2, 1, 1}}},
+    {{2, 17, 3, 4}, {{4}, {1, 3, 4}, {2, 1, 3, 4}}},
+    {{2, 17, 3, 4}, {{4}, {1, 3, 4}, {141, 1, 3, 4}}},
+    {{2, 1, 1, 3, 1}, {{1}, {1, 3, 4}, {2, 1, 3, 4}, {1, 1, 1, 1, 1}}},
+};
+
+static const std::vector<InferenceEngine::Precision> inputsPrecisions = {
+    InferenceEngine::Precision::FP64,
+    InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::FP16,
+    InferenceEngine::Precision::I32,
+    InferenceEngine::Precision::U32,
+    InferenceEngine::Precision::BOOL,
+};
+
+static const std::vector<ngraph::helpers::InputLayerType> secondInputTypes = {
+    ngraph::helpers::InputLayerType::CONSTANT,
+    ngraph::helpers::InputLayerType::PARAMETER,
+};
+
+static const std::map<std::string, std::string> additional_config = {};
```

Comment:
I will suggest to add struct with `static const` members:
```
struct ComparisionOpsData {
    const static std::map<std::vector<size_t>, std::vector<std::vector<size_t>>> inputShapes;
    const static std::vector<InferenceEngine::Precision> inputsPrecisions;
    cosnt static std::vector<ngraph::helpers::InputLayerType> secondInputTypes;
    const static std::map<std::string, std::string> additional_config;
```

and add initialization for this variables in cpp file.

Your proposal might not work as you expected. When you `include` file preprocessor more or less copy file content where include was called. Each cpp file where such heeders with static variables is called will have own copy of static variables.